modules/microshift-lvm-thin-volumes.adoc

```diff
@@ -46,4 +46,9 @@ device-classes: <2>
 [IMPORTANT]
 ====
 When multiple PVCs are created simultaneously, a race condition prevents LVMS from accurately tracking the allocated space and preserving the storage capacity for a device class. Use separate volume groups and device classes to protect the storage of highly dynamic workloads from each other.
+====
+
+[IMPORTANT]
+====
+When using thin provisioning, it is important that you monitor the storage pool and add more capacity as the available physical space runs out.
```

Comment:
Maybe adding a second line here. 
"Additionally, if there is available space within the Volume Group (VG), the storage pool could be also be configured to auto extent. See[ RHEL 9 documentation, Configuring and managing logical volumes, 11.2. Creating thinly-provisioned logical volumes](https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html-single/configuring_and_managing_logical_volumes/index#creating-thinly-provisioned-logical-volumes_creating-and-managing-thin-provisioned-volumes)" 

Then, maybe we need to enhance that RHEL doc based on some of the content from man. But not sure if this is still the ideal structure. 
From `man 7 lvmthin` and searching /thin_pool_autoextend:

> — Autoextending —
> dmeventd should be configured to extend thin pool LVs before all data space is used.  Warnings are emitted > through syslog when the use of a thin pool reaches 80%, 85%, 90% and 95%.  (See
> the  section "Data space exhaustion" for the effects of not extending a thin pool LV.)  The point at which > dmeventd extends thin pool LVs, and the amount are controlled with two configu‐
> ration settings:
> lvm.conf(5) thin_pool_autoextend_threshold
> is a percentage full value that defines when the thin pool LV should be extended.  Setting this to 100 disables > automatic extension.  The minimum value is 50.
> (..)
> Example
> If  thin_pool_autoextend_threshold  is  70  and thin_pool_autoextend_percent is 20, whenever a pool exceeds > 70% usage, it will be extended by another 20%.  For a 1G pool, using 700M will
trigger a resize to 1.2G. When the usage exceeds 840M, the pool will be extended to 1.44G, and so on.

Reply:
done. Important section is updated.